src/pages/zh-tw/install/manual.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 手動安裝 Astro
+description: 如何手動用 NPM、PNPM 或 Yarn 來安裝 Astro。
+layout: ~/layouts/MainLayout.astro
+setup: import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+i18nReady: true
+---
+準備好安裝 Astro 了嗎？跟隨我們的自動化或手動設定指南來開始吧。
+
+#### 事前準備
+
+- **Node.js** - `14.18.0`, `v16.12.0` 或更高版本。
+- **文字編輯器** - 我們推薦使用 [VS Code](https://code.visualstudio.com/) 以及我們的 [官方 Astro 延伸模組](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
+- **終端機** - Astro 可藉由命令行介面（CLI）來使用。
+
+<InstallGuideTabGroup />
+
+#### 安裝
+
+如果你偏好不使用我們自動化的 `create-astro` CLI 工具，你可以自己跟著下列指南來設定你的專案。
+
+## 1. 建立你的目錄
+
+建立一個空目錄並為你的專案命名，接著進到該目錄底下。
+
+```bash
+mkdir my-astro-project
+cd my-astro-project
+```
+當你進到你的新目錄後，建立你專案的 `package.json` 檔。這將會用來管理你專案的依賴模組包括 Astro。如果你不熟悉這個檔案的格式，可執行下列的指令來建立一個。
+
+```bash
+npm init --yes
+```
```

Comment:
We made this change to show the command for all three package managers:

````suggestion
<PackageManagerTabs>
  <Fragment slot="npm">
  ```shell
  npm init --yes
  ```
  </Fragment>
  <Fragment slot="pnpm">
  ```shell
  pnpm init 
  ```
  </Fragment>
  <Fragment slot="yarn">
  ```shell
  yarn init --yes
  ```
  </Fragment>
</PackageManagerTabs>
````